stringer.go

```diff
@@ -0,0 +1,30 @@
+package dig
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// String representation of the entire Container
+func (c Container) String() string {
+	b := &bytes.Buffer{}
+	fmt.Fprintln(b, "nodes: {")
+	for k, v := range c.nodes {
+		fmt.Fprintln(b, "\t", k, "->", v)
+	}
+	fmt.Fprintln(b, "}")
+
+	fmt.Fprintln(b, "cache: {")
+	for k, v := range c.cache {
+		fmt.Fprintln(b, "\t", k, "=>", v)
+	}
+	fmt.Fprintln(b, "}")
+
+	return b.String()
```

Comment:
Yeah, but it's not the intention to have a JSON output :) The target audience is for people using dig, not someone parsing the output of the graph.

That said, having it be *actual* JSON might be beneficial if we ever wanted to actually try and parse it for some reason (dunno, codegen or something)